### ISSUE TITLE
Is it possible to pre-record in go2rtc?

### ISSUE THREAD

**Issue by TokarevSergey** [AlexxIT/go2rtc#876]
Is it possible to pre-record in go2rtc?
&pre-record=5&
http://localhost:1984/api/stream.mp4?src=camera1&mp4=flac&pre-record=5&duration=5&filename=record.mp4

Thank you.

Record from H264 or H265 camera
Record from service call to the future. Doesn't support loopback.

mp4=flac - adds support PCM audio family
filename=record.mp4 - set name for downloaded file
service: telegram_bot.send_video
data:
  url: http://localhost:1984/api/stream.mp4?src=camera1&mp4=flac&duration=5&filename=record.mp4  # duration in seconds
  target: 123456789

**Comment by webbastelbude** [comment]
Yes i would be interested in this as well ! I would love to use go2rtc to get a clip from a camera for example if someone rings the doorbell or a motion sensor is being triggered.

**Comment by TokarevSergey** [comment]
It's a shame that few people are interested in this feature.
